qa/build.gradle

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+
+apply plugin: 'opensearch.testclusters'
+apply plugin: 'opensearch.build'
+apply plugin: 'opensearch.rest-test'
+
+// Disable a few tasks that come with build
+build.enabled = false
+integTest.enabled = false
+test.enabled = false
+assemble.enabled = false
+dependenciesInfo.enabled = false
+
+dependencies {
+    api "org.opensearch:opensearch:${opensearch_version}"
+    compileOnly "org.opensearch.plugin:opensearch-scripting-painless-spi:${versions.opensearch}"
+    api group: 'commons-lang', name: 'commons-lang', version: '2.6'
+
+    api "org.apache.logging.log4j:log4j-api:${versions.log4j}"
+    api "org.apache.logging.log4j:log4j-core:${versions.log4j}"
+
+    testImplementation "org.opensearch.test:framework:${opensearch_version}"
+    testImplementation(testFixtures(rootProject))
+}
```

Comment:
Should this be wrapped with 

```
subprojects {

}

``` 

to apply to all subprojects?

Reply:
I don't think we need to do it because this line of code will apply the dependencies to sub-projects
```
apply from : "$rootDir/qa/build.gradle"
```
Also, if it doesn't work. The existing tests would have failed.